data/strings/strings.txt

```diff
@@ -809,6 +809,99 @@
+    lv = Ierobežota precizitāte
+    mr = मर्यादित अचूकता
+    nb = Begrenset nøyaktighet
+    nl = Beperkte nauwkeurigheid
+    pl = Ograniczona dokładność
+    pt = Precisão limitada
+    ro = Acuratețe limitată
+    ru = Точность местоположения ограничена
+    sk = Obmedzená presnosť
+    sv = Begränsad noggrannhet
+    sw = Usahihi mdogo
+    th = ความแม่นยำ จำกัด
+    tr = Sınırlı Doğruluk
+    uk = Обмежена точність
+    vi = Độ chính xác hạn chế
+    zh-Hans = 精度有限
+    zh-Hant = 精度有限
+
+  [precise_location_is_disabled_long_text]
+    comment = A dialog text, that warns a user that Precise Location is disabled and suggests to turn it on
+    tags = android
+    en = To ensure accurate navigation enable Precise Location in settings.
+    af = Om 'n akkurate navigasie te verseker, maak dit 'n presiese ligging in instellings moontlik.
+    ar = لضمان التنقل الدقيق، قم بتمكين الموقع الدقيق في الإعدادات.
+    az = Dəqiq naviqasiyanı təmin etmək üçün parametrlərdə dəqiq bir yer təmin edin.
+    be = Для забеспячэння дакладнай навігацыі ўключыце дакладнае месцазнаходжанне ў наладах.
+    bg = За да осигурите точна навигация, активирайте функцията "Точно местоположение" в настройките.
+    ca = Per assegurar la navegació precisa, activeu la ubicació precisa en la configuració.
+    cs = Chcete-li zajistit přesnou navigaci, povolte v nastavení možnost Přesná poloha.
+    da = For at sikre nøjagtig navigation skal du aktivere Præcis placering i indstillingerne.
+    de = Um eine genaue Navigation zu gewährleisten, aktivieren Sie in den Einstellungen die Option Genauen Standort verwenden.
+    el = Για να εξασφαλίσετε ακριβή πλοήγηση, ενεργοποιήστε την Ακριβής τοποθεσία στις ρυθμίσεις.
+    es = Para garantizar una navegación más exacta, active la opción de ubicación precisa en los ajustes.
+    et = Täpse navigeerimise tagamiseks lülitage seadetes sisse Täpne asukoht.
+    eu = Nabigazio zehatzagoa ziurtatzeko, gaitu kokapen zehatza ezarpenetan.
+    fa = برای اطمینان از دقیق ناوبری ، موقعیت مکانی دقیق را در تنظیمات فعال کنید.
+    fi = Tarkan navigoinnin varmistamiseksi ota Tarkka sijainti käyttöön asetuksissa.
+    fr = Pour garantir une navigation précise, activez l'option Localisation précise dans les paramètres.
+    he = כדי להבטיח ניווט מדויק לאפשר מיקום מדויק בהגדרות.
+    hi = सटीक नेविगेशन सुनिश्चित करने के लिए सेटिंग्स में सटीक स्थान सक्षम करें।
+    hu = A pontos navigáció érdekében engedélyezze a beállítások között a Pontos helymeghatározás funkciót.
+    id = Untuk memastikan navigasi yang akurat, aktifkan Lokasi Tepat dalam pengaturan.
+    it = Per garantire una navigazione accurata, attivi la funzione Posizione precisa nelle impostazioni.
+    ja = 正確なナビゲーションを実現するには、設定で「正確な位置情報」を有効にしてください。
+    ko = 정확한 탐색을 위해 설정에서 정확한 위치를 활성화합니다.
+    lt = Norėdami užtikrinti tikslią navigaciją, nustatymuose įjunkite funkciją Tiksli vieta.
+    lv = Lai nodrošinātu precīzu navigāciju, iestatījumos iespējojiet funkciju Precīza atrašanās vieta.
```

Comment:
Lai nodrošinātu pareizu navigāciju, ieslēdziet iestatījumos "Precīzo" atrašanās vietas noteikšanu.